services/github-service-impl/src/main/java/org/kontinuity/catapult/service/github/impl/kohsuke/KohsukeGitHubServiceImpl.java

```diff
@@ -166,6 +176,24 @@ public GitHubRepository createRepository(String repositoryName,
         return wrapped;
     }
 
+    @Override
+    public GitHubRepository createRepository(String repositoryName, String description, File path) throws IllegalArgumentException {
```

Comment:
I think we should rename this method signature to something like `push(File path, GithubRepository)`.